src/main/kotlin/no/nav/medlemskap/regler/v1/arbeidsforhold/HarForetaketMerEnn5AnsatteRegel.kt

```diff
@@ -21,7 +23,12 @@ class HarForetaketMerEnn5AnsatteRegel(
 ) : ArbeidsforholdRegel(regelId, ytelse, periode) {
 
     override fun operasjon(): Resultat {
-        if (arbeidsforhold.antallAnsatteHosArbeidsgivere(kontrollPeriodeForArbeidsforhold) finnesMindreEnn 6) {
+
+        val arbeidsforholdMedMinstEnArbeidsavtaleOver0Stillingsprosent = arbeidsforhold.arbeidsforholdForKontrollPeriodeMedStillingsprosentOver0(kontrollPeriodeForArbeidsforhold)
+
+        if (arbeidsforholdMedMinstEnArbeidsavtaleOver0Stillingsprosent.antallAnsatteHosArbeidsgivere(kontrollPeriodeForArbeidsforhold) finnesMindreEnn 6 &&
+            arbeidsforholdMedMinstEnArbeidsavtaleOver0Stillingsprosent.antallAnsatteHosArbeidsgiversJuridiskeEnheter(kontrollPeriodeForArbeidsforhold) finnesMindreEnn 6
```

Comment:
Kunne du ha gjort disse to om til variabler og gitt dem et fornuftig navn slik at det blir litt lettere å lese?

Reply:
Tenkte du at f.eks arbeidsforholdMedMinstEnArbeidsavtaleOver0Stillingsprosent.antallAnsatteHosArbeidsgivere(kontrollPeriodeForArbeidsforhold) skal i en variabel?

Reply:
Jeg tenkte begge uttrykkene (med funksjonkallene og boolean uttrykkene), slik at man slipper å lese så store uttrykk for å se hva if'en ser etter. Men du får se hva som gir mest mening